### DISCUSSION TITLE
Anybody know why I keep getting annoying e-mail with "[mallman/exiv2] Run failed: Release - main (7ebf2a1)" as the subject

### DISCUSSION THREAD

**Discussion by mallman** [Exiv2/exiv2#2258] [General]
I have a fork of exiv2, https://github.com/mallman/exiv2, and I keep getting e-mail about "Release: Some jobs were not successful". I don't understand. Anyone know why I'm getting this and how to turn it off? It's getting really, really, REALLY annoying.
Thank you.

**Comment by andry81** [comment]
https://github.com/mallman/exiv2/actions/workflows/release.yml

Some job fails to run. You can turn off e-mails from the Account settings: https://github.com/settings/notifications

**Reply by mallman** [reply]
> https://github.com/mallman/exiv2/actions/workflows/release.yml

I don't want to turn off notifications for everything, but from the above link I found it's possible to manually disable workflows. Hopefully this will disable the nightly workflows.

**Comment by kevinbackhouse** [comment]
You can turn off GitHub Actions for an individual repo: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository

**Comment by mallman** [comment]
Thanks for your help guys. This is the first time I've ever experienced this. Correct me if I'm wrong, but does this mean that anyone who forks the exiv2 project will be running the nightly release actions on their fork automatically? (If not, I must have done something to enable them...)

**Reply by andry81** [reply]
Yes, you might have pressed one big green button in the Actions pane. A forked repository does not run Actions automatically. You have to enable them explicitly.
